pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-build-backend = "poetry.masonry.api"
+build-backend = "poetry.core.masonry.api"
```

Comment:
This seems like its been incorrect for some time, and its not usually a problem, since its not normally used. But I was installing the package in development mode for testing, which was failing because this was not correctly set. 

Poetry documentation about what the value should be: 
https://python-poetry.org/docs/pyproject/#poetry-and-pep-517

Reply:
Yeah, I looked this up and saw that it changed between 1.3 and 1.4. 🤷🏽‍♂️